README.md

```diff
@@ -52,6 +52,16 @@ This repo has the following functionality implemented in basic form:
   - Create scheduled actions (using `node-schedule` which I'm not sure is the best way to go)
 - `evaluateExpression` module
 
+## Docker
+
+To build image edit `Docker/dockerise.sh` and
```

Comment:
What do you mean "edit" -- don't you mean "run"?
"To build image, run `Docker/dockerise.sh`..."

Reply:
there is a line after that with command line to run

Reply:
I know, I just didn't get what the word "edit" is in there for. What are we editing?

Reply:
I hope this is an improvement: 0b1225afe326f05b9b7f29b822f81e8935e2660f